### ISSUE TITLE
Broadcasting and scalars lead to numpy warning

### ISSUE THREAD

**Issue by mhasself** [arahlin/qpoint#2]
Sort of a two-part issue here...

(a) The docs for azel2bore say the argument vectors need to be arrays with the same shape... but in fact broadcasting *is* supported.  Would be great if this were officially documented.

(b) The broadcasting triggers a numpy warning if there's a mix of scalars and arrays.  For example, this code:
```
import qpoint
import numpy as np

qp = qpoint.QPoint()
az, el, t = 90., 50., 1570000000.

print('test 1')
Q = qp.azel2bore(az, el, None, None,
                 lon=160., lat=-22., ctime=t)

print('test 2')
az, el, t = [np.array([_x]) for _x in [az,el,t]]
Q = qp.azel2bore(az, el, None, None,
                 lon=160., lat=-22., ctime=t)
print('done')
```
yields this output:
```
test 1
test 2
/home/mhasse/.local/lib/python3.7/site-packages/qpoint/_libqpoint.py:707: FutureWarning: future versions will not create a writeable array from broadcast_array. Set the writable flag explicitly to avoid this warning.
  arg.flags['WRITEABLE'] << 2 |
done
```

**Comment by arahlin** [comment]
You're right, broadcasting is supported, I'll fix the documentation to reflect that.  However, because the broadcast arrays aren't written to anyway, we shouldn't care if they are `WRITEABLE`, so I think this warning is safe to ignore.  Are you ok with me just suppressing the warning?

**Comment by mhasself** [comment]
Ok... but it sort of sounds like it's going to break in the future, when writeable arrays are expected but cannot be provided.  (It will break due to the unsatisfiable expectation, not due to actually trying to write to it.)  I think I'd prefer to dodge the Warning than have it suppressed.

**Comment by arahlin** [comment]
Fair point.  I'll see if I can figure out how to fix it.

**Comment by arahlin** [comment]
Fixed in f5336a4
